Since wxEpollDispatcher::DoPoll() returns the number of triggered events,
wxEpollDispatcher::HasPending() must not check for exactly one.

Signed-off-by: Anders Larsen <al@alarsen.net>
---
 src/unix/epolldispatcher.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/unix/epolldispatcher.cpp b/src/unix/epolldispatcher.cpp
index b5b7a8c..75fa090 100644
--- a/src/unix/epolldispatcher.cpp
+++ b/src/unix/epolldispatcher.cpp
@@ -194,7 +194,7 @@ wxEpollDispatcher::DoPoll(epoll_event *events, int numEvents, int timeout) const
 bool wxEpollDispatcher::HasPending() const
 {
     epoll_event event;
-    return DoPoll(&event, 1, 0) == 1;
+    return DoPoll(&event, 1, 0) >= 1;
 }
 
 int wxEpollDispatcher::Dispatch(int timeout)
-- 
1.5.4.3

